src/ngx_http_lua_shdict.c

```diff
@@ -1363,8 +1392,21 @@ ngx_http_lua_shdict_incr(lua_State *L)
             return 2;
         }
 
-        /* add value */
-        num = value + init;
+
+        if (flags == NGX_HTTP_LUA_SHDICT_INCR) {
+            /* add value */
+            num = value + init;
+        } else /* decr */ {
```

Comment:
I'll be happy to update these for style, but I need some clarification. The `}` is lined up with the `if` several lines above; are you saying this should be further indented? The style as is seems to align with existing `if... else` blocks in the codebase, so apologies if I'm missing something.